### PR TITLE
fix header layout in IE

### LINKED ISSUE #309: Header item distribution
Split from #230

The elements of the header aren't distributed the same in IE11 as they are in other browsers.

logo-placement-homepage

## Plan
- [x] investigate and fix... more..

## Acceptance Criteria
- [ ] The layout of the header looks the same as other browsers, or is decided to be OK as is

### PR DESCRIPTION
**Pull request by falconx**
Fixes #309

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -74,7 +78,7 @@ const Nav = () => (
         width: 100%;
       }
 
-      .list li + li {
+      .list > li + li {
```
Comment (katjam) [inline src/components/Header.js]: This will also have been affecting other browsers.

**Comment by falconx** [comment]
@katjam I'm not sure what your point is? I'd rather make apply fixes to make it work everywhere, not target IE specifically.

**Comment by katjam** [comment]
@falconx The review comment was in context of this issue #309 which specifically addressed a bug with ie11. We were tlking about whether or not there was another fix that needed testing or whether this was a preventative measure. 

Maybe add an AC if it fixes another layout problem.

**Comment by falconx** [comment]
@katjam it's all for the same fix :slightly_smiling_face

I've updated the AC to read "header" instead of "nav" to cover everything, just in case.

> The layout of the header looks the same as other browsers, or is decided to be OK as is

**Pull request merged**